package/CHANGELOG

```diff
@@ -14,11 +14,13 @@ The rules for this file:
 
 
 -------------------------------------------------------------------------------
-??/??/?? IAlibay, ChiahsinChu, RMeli, tanishy7777, talagayev
+??/??/?? IAlibay, ChiahsinChu, RMeli, tanishy7777, talagayev,
+         tyler.je.reddy
```

Comment:
Typo on gh uname?

Reply:
I think I've used them interchangeably, so not sure it matters much:

- `git grep -E -i "tyler\.je\.reddy" | wc -l` -> `13`
- `git grep -E -i "tylerjereddy" | wc -l ` -> `9`

Reply:
I don't see any such stipulation at the top of the `CHANGELOG`, and my entries as an author predate the development of MDAnalysis on GitHub anyway.

Nonetheless, I've pushed in a commit to remove the two periods to avoid any further time wasting on it.